components/SponsorsSection.js

```diff
@@ -54,10 +56,10 @@ export default function TicketsSection () {
   )
 }
 
-function ItemSponsor ({ src, type }) {
+function ItemSponsor ({ name, src, type }) {
   return (
     <div className={`sponsor-item-${type}`}>
-      <img src={src} width='100%' />
+      <img alt={`${name}`} aria-hidden src={src} width='100%' />
```

Comment:
Since we add an alt text now, we don’t need to use `aria-hidden` attribute.

p.s. Adding an `aria-hidden` to the image in the Speakers section is a good idea though 😃